.templates/template_location_v2.json

```diff
@@ -2,143 +2,143 @@
   {
     "repoPath": "https://github.com/GoogleCloudPlatform/cloud-code-samples.git",
     "directoryPath": "python/python-hello-world",
-    "templatePath": "python-hello-world",
+    "templatePath": "hello-world",
```

Comment:
This is not going to address the "issue" with the short/long names. This field is a path within the repo to the code. It's not visible to the users but is used for the extension to determine where code of the corresponding sample sits. If you want to change it across the samples, you also need to rename folders, update yamls for deployment and adjust the CI definitions. 

I personally don't think it worth it because we show all the samples for all the languages in the drop down where developers chooses to create a new app. So they might use the drop down twice or more for different languages and then get confused on which sample was cloned. However if the team decides this is the right thing to do for the product, I'll review it and merge when it's ready.

Reply:
@simonz130 - I'm not sure I understand your feedback.  VS Code uses the folder on disk as the "project name" so shortening the default folder name will thus shorten the name of the project.  The directory path is the path into the repository is it not?

Reply:
Did you mean this, @simonz130 ?
![image](https://user-images.githubusercontent.com/4393177/88222328-a8b6f500-cc1a-11ea-95ab-f82ce9a76dbf.png)

Reply:
@chairy, yes - this across all of the languages and samples.
@kschaab - are you saying that "templatePath" field is used only to determine the suggested folder where developers will have the template cloned? If so, what will happen if developers will try templates in two different languages for the same deployment platform? Are extensions checking on use-case of existing folder to offer a different folder (like hello-world1, hello-world2, etc.)? Is it the case in both IntelliJ and VSCode?

Reply:
@etanshaul - could you take a look from IntelliJ perspective too?

Reply:
@simonz130 if a user specifies two samples that are the same name they will be presented with a numeric suffix to differentiate them by default.  This only affects the default path that we present to the user, the user is free to type in any path that they would like to save to when we save the sample.

Reply:
OK, in that case I'll merge it - at least it will make the default saving directory proposal shorter.